Packs/HackerOne/pack_metadata.json

```diff
@@ -10,14 +10,9 @@
     "categories": [
         "Vulnerability Management"
     ],
-    "tags": [
-        "Security"
-    ],
-    "useCases": [
-        "Vulnerability Management"
-    ],
+    "tags": [],
+    "useCases": [],
     "keywords": [
-        "HackerOne",
```

Comment:
Why is this removed?

Reply:
keywords are added to the pack name words, so it is a redundancy.

Reply:
BTW, the pack description is also used in the search bar.